micro_http/src/request.rs

```diff
@@ -8,11 +8,11 @@ pub use common::RequestError;
 use common::{Body, Method, Version};
 use headers::Headers;
 
-// Helper functions used for parsing the HTTP Request.
-// Finds the first occurrence of the sequence in a byte slice and returns its
-// position in the slice.
-// If no occurrence is found, it returns `None`.
-fn find(bytes: &[u8], sequence: &[u8]) -> Option<usize> {
+/// Helper functions used for parsing the HTTP Request.
```

Comment:
Since this is public documentation now, the first comment line should be removed as it makes sense only in the context of this file. It's also just on helper function (you are saying helper functions in your comment).
Then, you should format the documentation as follows:
- first line (summary)
- space
- description

One suggestion:
```rust
\\\ Finds the first occurence of `sequence` in the `bytes` slice.
\\\
\\\ Returns the starting position of the `sequence` in `bytes` or `None` if the `sequence` is not found.
```